src/data/faq.json

```diff
@@ -595,7 +595,7 @@
                             "Yes of course, that is possible.",
                             "<b>Where can the Event QR code be generated?</b>",
                             "The QR code can be generated directly in the app or <a href='../eventregistration' target='_self' rel='noopener noreferrer'>here on the website</a>  and then printed as a PDF or scanned directly from the app.",
-                            "<b>Can Event QR codes from other apps be scanned with the CWA?</b>",
+                            "<b>Can Event QR codes from other apps (e.&nbsp;g. the 'luca' app) be scanned with the CWA?</b>",

```

Comment:
Also here, no `nbsp`

Reply:
Done in https://github.com/corona-warn-app/cwa-website/commit/6526b70462a14205e6a81858dfc274ab6eb3ea5a, too.